src/libraries/tests.proj

```diff
@@ -278,6 +278,19 @@
     <ProjectExclusions Include="$(MSBuildThisFileDirectory)System.Reflection/tests/System.Reflection.Tests.csproj" />
     <ProjectExclusions Include="$(MSBuildThisFileDirectory)System.Threading.Channels/tests/System.Threading.Channels.Tests.csproj" />
 
+    <!-- 
+      System.Formats.Cbor crashes on tvOS 
+    -->
+    <ProjectExclusions Include="$(MSBuildThisFileDirectory)System.Formats.Cbor\tests\System.Formats.Cbor.Tests.csproj" />
+
+    <!--
+      Test apps that are too large and take too long to build 
+      Keep here until aggressive trimming targets can work on helix.
+    -->
+    <ProjectExclusions Include="$(MSBuildThisFileDirectory)System.Text.RegularExpressions\tests\UnitTests\System.Text.RegularExpressions.Tests.csproj" />
+    <ProjectExclusions Include="$(MSBuildThisFileDirectory)System.Text.Json\tests\System.Text.Json.SourceGeneration.Tests\System.Text.Json.SourceGeneration.Roslyn4.0.Tests.csproj" />
+
+
```

Comment:
Are these new failures? Are there issues associated that we can put in the comments?

Reply:
If it's just links that we need to add, then we can do that in a follow up PR.

Reply:
There will be.  I would rather address it in https://github.com/dotnet/runtime/pull/67861